### PR TITLE
Task-58742: Display dynamically onedrive and gdrive connectors

### PR DESCRIPTION
**Pull request by abenmbarek**
Prior to this change, we display statically onedrive and gdrive connectors directly from ECMS codebase. After this commit we ensure that :

- ECMS codebase implements only the cloud drive connectors framework with the cloud drive setting drawer portlet
- Cloud-drive-connectors addon codebase implements onedrive and gdrive connectors and injects them into the cloud drive setting drawer portlet.

### CODE REVIEW

**Review by azayati — CHANGES_REQUESTED** [review]
- You should move Google Drive.png and One Drive.png to the addon
- You should try to remove the whole modules **ecms/ext/clouddrives-gdrive/webapp/** and **ecms/ext/clouddrives-onedrive/webapp/** after verifying where the icons are used

File: apps/portlet-clouddrives/src/main/webapp/WEB-INF/gatein-resources.xml [hunk 1]
```diff
@@ -48,12 +48,12 @@
...
         <module>cloudDriveUtils</module>
       </depends>
-      <depends>
+<!--      <depends>
         <module>cloudDrive_gdrive</module>
       </depends>
       <depends>
         <module>cloudDrive_onedrive</module>
-      </depends>
+      </depends>-->
```
Comment (azayati) [inline apps/portlet-clouddrives/src/main/webapp/WEB-INF/gatein-resources.xml]: this comment should be removed and you should verify if CloudDrivePortlet need to be removed.

File: apps/portlet-clouddrives/src/main/webapp/WEB-INF/gatein-resources.xml [hunk 2]
```diff
@@ -78,6 +78,18 @@
       <depends>
         <module>commonVueComponents</module>
       </depends>
+      <depends>
+        <module>cloudDriveUtils</module>
```
Comment (azayati) [inline apps/portlet-clouddrives/src/main/webapp/WEB-INF/gatein-resources.xml:76]: is it needed to add cloudDriveUtils as dependency for the CloudDriveSettingsPortlet ?

File: apps/portlet-clouddrives/src/main/webapp/WEB-INF/gatein-resources.xml [hunk 3]
```diff
@@ -78,6 +78,18 @@
...
+      </depends>
+<!--      <depends>
+        <module>cloudDrive_gdrive</module>
+      </depends>
+      <depends>
+        <module>cloudDrive_onedrive</module>
+      </depends>-->
```
Comment (azayati) [inline apps/portlet-clouddrives/src/main/webapp/WEB-INF/gatein-resources.xml]: remove this comment

File: apps/portlet-clouddrives/src/main/webapp/WEB-INF/gatein-resources.xml [hunk 4]
```diff
@@ -78,6 +78,18 @@
...
+      <depends>
+        <module>cloudDrives</module>
```
Comment (azayati) [inline apps/portlet-clouddrives/src/main/webapp/WEB-INF/gatein-resources.xml:79]: s it needed to add cloudDrives as dependency for the CloudDriveSettingsPortlet ?

File: apps/portlet-clouddrives/src/main/webapp/skin/clouddrive.css [hunk 5]
```diff
@@ -391,4 +391,14 @@
 #connectCloudApp .cloudDriveListItem__title {
 	font-size: 14px;
 }
+
+#cloudDriveSettingsDrawer .noEnabledConnectors .uiIconCloudDriveConnector {
+	font-size: 52px;
+	height: 52px;
+	color: #4d5466 !important;
```
Comment (azayati) [inline apps/portlet-clouddrives/src/main/webapp/skin/clouddrive.css]: use appropriate color variable please and ensure file indentation

File: core/webui-clouddrives/src/main/resources/locale/clouddrive/CloudDrive_en.xml [hunk 6]
```diff
@@ -110,11 +110,10 @@
     </label>
     <drawer>
       <title>Connect your personal drives</title>
-      <google>Google Drive</google>
-      <onedrive>One Drive</onedrive>
       <button>
         <connect>Connect</connect>
       </button>
+      <NoConnector>No personal drive connector available</NoConnector>
```
Comment (azayati) [inline core/webui-clouddrives/src/main/resources/locale/clouddrive/CloudDrive_en.xml]:
```suggestion
      <noConnector>No personal drive connector available</noConnector>
```

File: ext/clouddrives-gdrive/webapp/src/main/webapp/WEB-INF/gatein-resources.xml [hunk 7]
```diff
@@ -24,17 +24,4 @@
     xsi:schemaLocation="http://www.gatein.org/xml/ns/gatein_resources_1_3 http://www.gatein.org/xml/ns/gatein_resources_1_3"
     xmlns="http://www.gatein.org/xml/ns/gatein_resources_1_3">
 
-  <!-- Javascript for Cloud Drive support of Google Drive -->
```
Comment (azayati) [inline ext/clouddrives-gdrive/webapp/src/main/webapp/WEB-INF/gatein-resources.xml:27]: the whole file should be removed IMO

File: ext/clouddrives-onedrive/webapp/src/main/webapp/WEB-INF/conf/configuration.xml [hunk 8]
```diff
@@ -24,103 +24,4 @@
   xsi:schemaLocation="http://www.exoplatform.org/xml/ns/kernel_1_2.xsd http://www.exoplatform.org/xml/ns/kernel_1_2.xsd"
   xmlns="http://www.exoplatform.org/xml/ns/kernel_1_2.xsd">
 
-  <!-- One Drive namespace for use within JCR -->
```
Comment (azayati) [inline ext/clouddrives-onedrive/webapp/src/main/webapp/WEB-INF/conf/configuration.xml:27]: the whole file should be removed IMO

File: ext/clouddrives-gdrive/webapp/src/main/webapp/WEB-INF/conf/configuration.xml [hunk 9]
```diff
@@ -24,103 +24,4 @@
   xsi:schemaLocation="http://www.exoplatform.org/xml/ns/kernel_1_2.xsd http://www.exoplatform.org/xml/ns/kernel_1_2.xsd"
   xmlns="http://www.exoplatform.org/xml/ns/kernel_1_2.xsd">
 
-  <!-- Google Drive namespace for use within JCR -->
```
Comment (azayati) [inline ext/clouddrives-gdrive/webapp/src/main/webapp/WEB-INF/conf/configuration.xml:27]: the whole file should be removed IMO

File: ext/clouddrives-onedrive/webapp/src/main/webapp/WEB-INF/gatein-resources.xml [hunk 10]
```diff
@@ -22,40 +22,4 @@
   xsi:schemaLocation="http://www.exoplatform.org/xml/ns/gatein_resources_1_4 http://www.exoplatform.org/xml/ns/gatein_resources_1_4"
   xmlns="http://www.exoplatform.org/xml/ns/gatein_resources_1_4">
 
-  <!-- Javascript for Cloud Drive support of One Drive -->
```
Comment (azayati) [inline ext/clouddrives-onedrive/webapp/src/main/webapp/WEB-INF/gatein-resources.xml:25]: the whole file should be removed IMO

**Review by azayati — CHANGES_REQUESTED** [review]

File: apps/portlet-clouddrives/src/main/webapp/WEB-INF/gatein-resources.xml (hunk 2 above)
Comment (abenmbarek) [reply apps/portlet-clouddrives/src/main/webapp/WEB-INF/gatein-resources.xml:76]: yep it's needed to use the getUserDrive function under /js/cloudDriveService.js

File: apps/portlet-clouddrives/src/main/webapp/WEB-INF/gatein-resources.xml (hunk 4 above)
Comment (abenmbarek) [reply apps/portlet-clouddrives/src/main/webapp/WEB-INF/gatein-resources.xml:79]: yep it's used in every cloudDrive config js files

File: apps/portlet-clouddrives/src/main/webapp/skin/clouddrive.css (hunk 5 above)
Comment (abenmbarek) [reply apps/portlet-clouddrives/src/main/webapp/skin/clouddrive.css]: this indentation is used in the whole file so i respected its indentation

Comment (abenmbarek) [reply apps/portlet-clouddrives/src/main/webapp/skin/clouddrive.css]: it's a css file so we cannot use color variables from variables.less

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**